lib/citgm.js

```diff
@@ -29,14 +30,38 @@ exports.windows = windows;
  * 5. Output the results.
  **/
 
-function find(app, context, next) {
-  which(app, function(err, resolved) {
+function findNode(context, next) {
+  which('node', function(err, resolved) {
     if (err) {
-      next(Error(app + ' not found in path!'));
+      npmWhich(__dirname)('node', function(err, resolved) {
+        if (err) {
+          next(Error('node not found in path!'));
+          return;
+        }
+
+        context.emit('data', 'verbose',
+          context.module.name + ' using-node',
+          resolved);
```

Comment:
not happy about this formatting, but that'll be fixed when we merge the prettier PR, so I'm kinda meh on spending time aligning stuff here, tbh